common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -148,6 +148,12 @@ div.problem {
       margin-top: $baseline;
     }
   }
+
+  span > label {
+    display: block;
+    margin-bottom: $baseline;
+    font: inherit;
+  }
```

Comment:
@chris-mike Can you suggest something which color to use here. This is the place where we're setting up color for newly added label.